app/components/services/services.html

```diff
@@ -93,6 +99,12 @@
                   </span>
                 </td>
                 <td>
+                  <a ng-if="service.Ports.length > 0 && swarmManagerIP" ng-repeat="p in service.Ports" class="image-tag" ng-href="http://{{swarmManagerIP}}:{{p.PublishedPort}}" target="_blank">
+                    <i class="fa fa-external-link" aria-hidden="true"></i> {{ p.PublishedPort }}:{{ p.TargetPort }}
+                  </a>
+                  <span ng-if="service.Ports.length == 0 || !swarmManagerIP" >-</span>
```

Comment:
Use the `===` operator for the comparison.

Somehow, this is not displayed anyway (the `-`). Could you fix this?

![portainer 2](https://cloud.githubusercontent.com/assets/5485061/24359279/2a74ccb2-1304-11e7-8155-98107dc582b4.png)